tests/Feature/Http/Controllers/TradeControllerTest.php

```diff
@@ -35,7 +35,18 @@ public function setUp(): void
      */
     public function index_displays_view()
     {
-        $trades = Trade::factory()->count(3)->create();
+        $trades = Trade::factory(3)
+            ->has(Item::factory(3))
+            ->create();
```

Comment:
This isnt the state a trade would be in; basically a trade will post an `item` and have offers in the `items` relation (note the s) so you'd need to define the specific relationships your assigning the items too, you can find how that works [here in the docs](https://laravel.com/docs/8.x/database-testing#has-many-relationships), note the second code block where they specify the relationship name